ansible/vars/icds/icds_public.yml

```diff
@@ -0,0 +1,259 @@
+SITE_HOST: 'www.icds-cas.gov.in'
+#SITE_HOST: 'cas.commcarehq.org'
+CAS_SITE_HOST: 'cas.commcarehq.org'
+#NO_WWW_SITE_HOST: 'cas.commcarehq.org'
+NO_WWW_SITE_HOST: 'icds-cas.gov.in'
+deploy_env: icds
+tableau_server: 10.247.24.11:80
+
+# these 2 must be replaced with the real icds cert when we get it from NIC
+nginx_combined_cert_file: '../config/{{ deploy_env}}/ssl/cas.commcarehq.org.combined.crt'
+apache2_key_file: '../config/{{ deploy_env }}/ssl/cas.commcarehq.org.key'
+
+cas_nginx_combined_cert_file: '../config/{{ deploy_env}}/ssl/cas.commcarehq.org.combined.crt'
+cas_key_file: '../config/{{ deploy_env }}/ssl/cas.commcarehq.org.key'
+
+active_sites:
+  cchq_ssl: True
+  cchq_redirect: True
+  cchq_http: True
+  cchq_http_redirect: True
+  cchq_http_j2me: False
+  commtrack_ssl: False
+  commtrack_http: False
+  hq_status: False
+  tableau: False
+  icds_tableau: False
+  dhis: False
+  wiki: False
+  wiki_http: False
+  motech: False
+  motech1: False
+  motech2: False
+  riakcs: True
+  cas_ssl: True
+
+riak_ring_size: 128
+
+proxy_type: 'nginx'
+primary_ssl_env: "icds"
```

Comment:
I think it would be easier if primary_ssl_env is set to `cas`. Otherwise we're going to have to add another conditional for enikshay when that gets moved as well and that if statement is going to become longer and longer